CONTRIBUTING.md

```diff
@@ -1,12 +1,13 @@
 # [<img src="https://avatars1.githubusercontent.com/u/3771091?v=3&s=30">](https://github.com/gonum) Contribution Guidelines
 #### Introduction
 
-The Gonum project provides numerical computing and data science tools for the Go programming language.  This document is for anyone who is contributing or interested in contributing.
+The Gonum project provides general purpose numerical computing and data science libraries for the Go programming language.  This document is for anyone who is contributing or interested in contributing.
```

Comment:
Done.  I linked it with the first sentence to make it flow a little better to me.